samples/subsys/tracing/prj_percepio.conf

```diff
@@ -4,3 +4,4 @@ CONFIG_MP_MAX_NUM_CPUS=1
 CONFIG_DEBUG_THREAD_INFO=y
 CONFIG_PERCEPIO_TRACERECORDER=y
 CONFIG_TRACING=y
+CONFIG_TRACING_GPIO=n
```

Comment:
why are these required? wouldn't they just compile to nothing if they are not implemented by this backends?

Reply:
Yeah that makes sense - we will fix that

Reply:
@ycsin - does this look OK now?